README.md

```diff
@@ -36,8 +36,8 @@ NOTE: The API is unstable any may be subject to changes until it reaches 1.0.
 
 ## Install
 
-```
-$ pip install pycobertura
+```bash
+pip install pycobertura
```

Comment:
Should this command be `pip3 install pycobertura`?

Reply:
I do not think that it is really necessary, especially as if you only have Python 3 installed the commands `python` and `pip` should work without any configuration. Also normally people work with virtual environments these commands also will always work as they should. But if you want to make clear that Pycobertura only supports Python3 we could also change it.